docs/editor/extension-marketplace.md

```diff
@@ -334,9 +334,16 @@ You may see this error if your machine is going through a proxy server to access
 
 ### Can I download an extension directly from the Marketplace?
 
-Some users prefer to download an extension once from the Marketplace and then install it multiple times from a local share. This is useful when there are connectivity concerns or if your development team wants to use a fixed set of extensions.
+Some users prefer to download an extension once from the Marketplace and then install it to multiple VS Code instances from a local share. This is useful when there are connectivity concerns or if your development team wants to use a fixed set of extensions.
 
-To download an extension, navigate to the details page for the specific extension within the [Marketplace](https://marketplace.visualstudio.com/vscode). On that page, there is a **Download Extension** link in the **Resources** section, which is located on the right-hand side of the page.
+To download an extension, use the following command option:
+```
+code --download-extension <UniqueExtensionID> --location <Path>
+
+Example:
+code --download-extension ms-python.python --location ~/vscode/testing
+```
+You can copy the extension ID from the extension details page ![Copy Extension ID](./images/extension-marketplace/copy-extension-id.png).
```

Comment:
Let's remove the "you can copy the extension ID" sentence. This does not have to be super easy.

Also let's add a note, something like "--download-extension is available in VS Code Insiders, and will be in VS Code Stable in December."

Reply:
@isidorn I thought the plan was to release this guidance with the December release anyway?

Reply:
I would release this guidance before, so that Marketplace can go ahead with their plan of removing the download URL before the Stable VS Code is out.